File: android-app/app/src/main/java/com/example/campusconnect/model/RegisterRequest.java
package com.example.campusconnect.model;

public class RegisterRequest {

    private String email;
    private String username;
    private String password;

    public RegisterRequest(String email, String username, String password) {
        this.email = email;
        this.username = username;
        this.password = password;
    }

}

File: android-app/app/src/main/java/com/example/campusconnect/feature/auth/data/remote/request/RegisterRequest.kt
package com.example.campusconnect.feature.auth.data.remote.request

data class RegisterRequest(

    val username: String,

    val email: String,

    val password: String,

    val fullName: String,

    val courseId: Int,

    val admissionYear: Int,

    val gender: String,

    val dob: String,

    val rollNumber: String,

    val googleIdToken: String
)